### PR TITLE
Task3frontend

### PR DESCRIPTION
**Pull request by mgirar84**

### CODE REVIEW

File: css.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+}
+
+.collectionContainer2{
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+}
```
Comment (HiredMussel) [inline css.css:10]: This might be redundant - consider making these two the same class

File: css.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+.collectionItem{
+    border: solid 0.1em;
+    margin: 0.5em;
+    padding-left: 0.5em;
+    padding-right: 0.5em;
+    padding-bottom: 0.5em;
+}
```
Comment (HiredMussel) [inline css.css:19]: Play around with percentage widths at some point if you want to have three items to a row

File: index.php [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            <div class="collectionItem">
+                <h2>Collection Item 1</h2>
+                <img class="collectionImage" src="https://dbyvw4eroffpi.cloudfront.net/product-media/2L9M/1000/1000/Cannondale-SystemSix-Carbon-Ultegra-Di2-Disc-Road-Bike-2020.jpg"></img>
+                <ul class="informationList">
+                    <li>Name:</li>
+                    <li>Brand:</li>
+                    <li>Price:</li>
+                    <li>Class:</li>
+                </ul>
+            </div>
```
Comment (HiredMussel) [inline index.php:20]: Eventually I assume you'd be generating this from the database using PHP

**Review by HiredMussel — APPROVED** [review]

File: index.php (hunk 3 above)
Comment (mgirar84) [reply index.php:20]: correct, its placeholder for now

File: css.css (hunk 2 above)
Comment (mgirar84) [reply css.css:19]: adjusted to 33% and merged into one class

File: css.css (hunk 1 above)
Comment (mgirar84) [reply css.css:10]: complete

**Pull request merged**